specification/healthcareapis/resource-manager/readme.md

```diff
@@ -38,9 +38,18 @@ azure-arm: true
 These settings apply only when `--tag=package-2024-03` is specified on the command line.
 
 ```yaml $(tag) == 'package-2024-03'
+input-file:
+  - Microsoft.HealthcareApis/stable/2024-03-31/healthcare-apis.json
+```
+### Tag: package-2024-03
```

Comment:
`Tag: package-2024-03` is duplicated defined.
https://github.com/Azure/azure-rest-api-specs/pull/27981/checks?check_run_id=22104548809

Reply:
@ms-zhenhua, Since we have 2 API version in march, how do we want to put there. Should I only include the latest one? or can I include the day also in the tag